src/convertToObject.js

```diff
@@ -4,9 +4,26 @@
  * @param {string} sourceString
  *
  * @return {object}
+ *
  */
+const removeSpaces = (str) => str.trim();
+
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const styleObject = {};
+
+  sourceString
+    .split(';')
+    .map(removeSpaces)
+    .filter((el) => el.length > 0)
+    .reduce((obj, el) => {
+      const propStyle = el.split(':').map(removeSpaces);
```

Comment:
![image](https://github.com/mate-academy/js_style-to-object/assets/96409563/4f32bb29-2fd8-497a-a50c-e663d3eb162c)